### ISSUE TITLE
User stats page

### ISSUE THREAD

**Issue by zwrawr** [ludumdare/ludumdare#1469]
*  #1309 event by event rankings
* 
    > (18:38:26) blubberquark: I would like to see a histogram of my votes versus the histogram of other people's votes
    > (18:38:43) blubberquark: I want to know if I am a higher or lower ranker

**Comment by povrazor** [comment]
A stub is ready and waiting. :smile:

https://ldjam.com/user/zwrawr/stats

Router Page: `/src/com/page/user/stats/stats.js`

**Comment by zwrawr** [comment]
Rough mock up
![screenshot from 2017-12-10 07-59-13](https://user-images.githubusercontent.com/18352787/33803126-1c628bf4-dd80-11e7-847a-ed1365590eb7.png)

* Don't like the size of the graphs in the middle, maybe reduce there size a bit and centre them
* Need to get tooltips working for the graphs to make any sense

**Comment by local-minimum** [comment]
I would like a caption at the bottom of most graphs to explain what they actually show because right now I'm not sure what half of them mean.

But they're looking good.

I'm guessing here, but if all these graphs are about my results, it would also be nice to have what I think bubblerquark asked for. How the distribution of votes that I cast look like and other metrics about how I do during the voting period. When I vote during the 21 day period. And for those metrics, it would be nice to have it compare against the average behaviour of people on the site. Some of those graphs could encourage people to participate in voting.

But, since that makes two very distinct categories of information. It would be good if there was a clear heading / separator between the two.

**Comment by zwrawr** [comment]
I don't think we actually have the average break down of how many `5*` vs `4*` ect votes.
I guess that is something `magic.php` could do?

**Comment by sgstair** [comment]
it is obviously not something that should be shown during voting (as it could be used to correlate votes with users) - but it is pretty easy to build an API to provide this information.

**Comment by local-minimum** [comment]
Oh,  how you vote should probably only ever be visible to you.  Good point.

**Comment by povrazor** [comment]
I just want to add that from the mockup this looks pretty slick. :smile:
